website/layout/DocsLayout.js

```diff
@@ -48,6 +48,15 @@ var DocsLayout = React.createClass({
               {metadata.previous && <a className="docs-prev" href={'docs/' + metadata.previous + '.html#content'}>&larr; Prev</a>}
               {metadata.next && <a className="docs-next" href={'docs/' + metadata.next + '.html#content'}>Next &rarr;</a>}
             </div>
+            <div className="survey">
+              <div className="survey-image" />
+              <p>
+                We just had a lockdown to improve the React Native documentation. Your responses to this yes/no style survey will help us gauge whether we moved in the right direction with the improvements. Thank you!
```

Comment:
Is the term 'lockdown' used in a similar manner outside of FB? Maybe go with similar wording as the [blog post](https://github.com/lacker/react-native/blob/e5149c8c1fda8dbbb2ff0209303eca57447bf235/blog/2016-07-05-toward-better-documentation.md):

> "Recently we've been working hard to make the docs better based on your feedback. Your responses to this yes/no style survey will help us gauge whether we moved in the right direction with the improvements. Thank you!"